protocol/generated.proto

```diff
@@ -1,14 +1,3 @@
-// Copyright 2017 Sourced Technologies SL
```

Comment:
What's the reason to remove this?
Why is this removed from some files while left in other files?